<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>jpg Name Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>170</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>15</y>
      <width>131</width>
      <height>52</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>15</y>
      <width>131</width>
      <height>52</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_jpgList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>210</y>
      <width>411</width>
      <height>115</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_targetName">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>82</y>
      <width>112</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>85</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Name</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_gray">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>87</y>
      <width>96</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>133</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotate(degree)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_rotate">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>130</y>
      <width>112</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
